server.js

```diff
@@ -60,6 +60,7 @@ app.get('/logout', (req, res) => {
 })
 
 app.use("/api", require("./routes"));
+app.use("/users/user", require("./userRoutes"));
```

Comment:
No need to repeat /user/users, that creates unnecessary paths. /users is sufficient